make/containers.mk

```diff
@@ -31,88 +31,100 @@ BASE_IMAGE_cmctl-linux-s390x:=$($(BASE_IMAGE_TYPE)_BASE_IMAGE_s390x)
 BASE_IMAGE_cmctl-linux-ppc64le:=$($(BASE_IMAGE_TYPE)_BASE_IMAGE_ppc64le)
 BASE_IMAGE_cmctl-linux-arm:=$($(BASE_IMAGE_TYPE)_BASE_IMAGE_arm)
 
-bin/containers:
-	@mkdir -p $@
-
 .PHONY: all-containers
 all-containers: cert-manager-controller-linux cert-manager-webhook-linux cert-manager-acmesolver-linux cert-manager-cainjector-linux cert-manager-ctl-linux
```

Comment:
bug: I checked out this branch locally and got the following error:

```console
$ make -f make/Makefile all-containers
make/containers.mk:104: *** multiple target patterns.  Stop.
```

Reply:
My apology, I pushed a broken commit. It should be fixed now! 😇